### PR TITLE
Feature/main page updates

### PR DESCRIPTION
**Pull request by ljoks**
fixes minor things that were bugging me: 
- truncating long titles in the content tree
- max title length in the create title modal
- display the folder in the question preview

### CODE REVIEW

File: frontend/src/components/home-page/content-tree/content-tree.tsx [hunk 1]
```diff
@@ -322,7 +326,7 @@ const ContentTree = (): ReactElement => {
                                                 }
                                               >
                                                 <div className="title">
-                                                  {question.title}
+                                                  {truncate(question.title)}
```
Comment (weining-li) [inline frontend/src/components/home-page/content-tree/content-tree.tsx:329]: add
```
    text-overflow: ellipsis;
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
```
to the `title` class
```
  .title,
  .type {
    overflow: hidden;
    font-size: 1.25rem;
  }
```
to have ellipsis replace text longer than 2 lines.

File: frontend/src/components/creator-module/creator-edit/creator-edit.tsx [hunk 2]
```diff
@@ -60,6 +63,7 @@ const CreatorEdit = ({ newQuestion, setNewQuestion }: Prop): ReactElement => {
             className="question-title-input"
             placeholder="eg: Question 1 Title"
             defaultValue={previewQuestion.title}
+            value={newQuestion.title}
```
Comment (weining-li) [inline frontend/src/components/creator-module/creator-edit/creator-edit.tsx:66]: add `maxLength` attribute to the input element to limit characters.
for example `maxlength={50}`

**Review by weining-li — CHANGES_REQUESTED** [review]
:cactus:

**Review by weining-li — APPROVED** [review]
:u5272:

File: frontend/src/components/home-page/question-preview/question-preview.scss [hunk 3]
```diff
@@ -13,6 +13,20 @@
     margin: 3rem 1rem 0 1rem;
   }
 
+  &__folder {
+    // border: 1px solid red;
```
Comment (Goff-Davis) [inline frontend/src/components/home-page/question-preview/question-preview.scss:17]: No need to keep commented code

**Review by Goff-Davis — APPROVED** [review]
Minor thing but looks good to me

**Review by Goff-Davis — APPROVED** [review]
LGTM

**Pull request merged**